tests/commands/test_changelog_command.py

```diff
@@ -1210,3 +1216,160 @@ def test_changelog_prerelease_rev_with_use_version_type_semver(
     out, _ = capsys.readouterr()
 
     file_regression.check(out, extension=".second-prerelease.md")
+
+
+@pytest.mark.parametrize(
```

Comment:
The tests here look similar to the ones in `tests/commands/test_bump_command.py`. Should we reduplicate them? This doesn't have to be done in this PR